components/feature/autofill/src/main/res/layout/mozac_feature_autofill_preference.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.appcompat.widget.SwitchCompat xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/switch_widget"
+    android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    android:clickable="false"
```

Comment:
Should this be true or was it meant to be changed programmatically?

Reply:
Same for `focusable`?

Reply:
Yeah, this was on purpose, we do not want this individual view be clickable/focusable, but the whole preference entry.